.github/workflows/build_docker.yml

```diff
@@ -77,12 +77,11 @@ jobs:
         npm run build
 
     - name: Commit report
-      if: steps.cache-frontend-dist.outputs.cache-hit != 'true'
```

Comment:
Do we still want this change?

Reply:
We do, the cache recovery is equivalent to building the dist, not building and committing; 
if we revert a change we still want it to build, but it doesn't if we have this check in.